modules/virt-subscribing-cli.adoc

```diff
@@ -39,10 +39,13 @@ spec:
   sourceNamespace: openshift-marketplace
   name: kubevirt-hyperconverged
   startingCSV: kubevirt-hyperconverged-operator.v{HCOVersion}
-  channel: "{VirtVersion}"
+  channel: "stable" <1>
 ----
+<1> Using the `stable` channel ensures that you will install the version of
```

Comment:
Definite nit but I think we can drop 'will'
Here and in following two instances.

Reply:
I wasn't sure if this was an exception to that rule or not. 😄 Removed.